src/Language/GraphQL/Draft/Syntax.hs

```diff
@@ -320,6 +320,7 @@ data Value var
   | VInt Integer
   | VFloat Scientific
   | VString Text
+  | VBlockString Text
```

Comment:
I was thinking instead of adding a new constructor `VBlockString` if we could extend the `VString` type in the following manner.

```haskell
data StringType = StringCharacter | BlockStringCharacter

data Value var
  = VVariable var
  | VNull
  | VInt Integer
  | VFloat Scientific
  | VString StringType Text
  | VBoolean Bool
  | VEnum EnumValue
  | VList [Value var]
  | VObject (HashMap Name (Value var))
```

This way both the string types are clubbed together and I think except the parsing and printing bit, both types of string will be processed in the same way.